Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -368,4 +372,32 @@ extension ChatChannelListItemView {
     var isLastMessageVoiceRecording: Bool {
         content?.channel.previewMessage?.voiceRecordingAttachments.isEmpty == false && typingUserString == nil
     }
+
+    /// The message preview text in case it contains attachments.
+    /// - Parameter previewMessage: The preview message of the channel.
+    /// - Returns: A string representing the message preview text.
+    func attachmentPreviewText(for previewMessage: ChatMessage) -> String? {
+        guard let attachment = previewMessage.allAttachments.first else {
+            return nil
+        }
+        let text = previewMessage.textContent ?? previewMessage.text
+        switch attachment.type {
+        case .audio:
+            return "🎧 \(text.isEmpty ? "Audio" : text)"
```

Comment:
All the texts here should be localized, not hardcoded to English.

Reply:
Good catch 👍

Reply:
Solved ✅